From 490d58cbff669af3c1639b4d6df9151a064e964e Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 9 Jul 2018 02:15:39 +0300
Subject: [PATCH 204/249] kuntao: Update custom TWRP link

* TWRP 3.2.2 built from latest lineage trees
* Built with toybox and excluding TWRP app

Change-Id: I8ff9173eaa45b480a40f892dd95b8124c7819947
---
 _data/devices/kuntao.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_data/devices/kuntao.yml b/_data/devices/kuntao.yml
index ea84fb3..fa19543 100644
--- a/_data/devices/kuntao.yml
+++ b/_data/devices/kuntao.yml
@@ -11,7 +11,7 @@ cpu: Cortex-A53
 cpu_cores: '8'
 cpu_freq: 2.0 GHz
 current_branch: 15.1
-custom_twrp_link: https://androidfilehost.com/?fid=890129502657588434
+custom_twrp_link: https://androidfilehost.com/?fid=5862345805528044659
 depth: 8.3 mm (0.33 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 506
-- 
2.17.1

